ipapython/install/cli.py

```diff
@@ -136,22 +136,21 @@ def add_options(cls, parser, positional=False):
                 help="unattended (un)installation never prompts the user",
             )
 
-        basic_group = optparse.OptionGroup(parser, "basic options")
-
         groups = collections.OrderedDict()
-        groups[None] = basic_group
+        groups[None] = parser
```

Comment:
Oh, I see, so this will create some set of basic options. Please, could you add a comment there explaining it so that it's very clear?